packages/next/src/build/webpack-config.ts

```diff
@@ -492,7 +492,7 @@ function getModularizeImportAliases(packages: string[]) {
 
       for (const field of mainFields) {
         if (descriptionFileData.hasOwnProperty(field)) {
-          aliases[pkg] = path.join(
+          aliases[pkg + '$'] = path.join(
```

Comment:
This is for regex termination, right?

Reply:
I think it's webpack alias syntax https://webpack.js.org/configuration/resolve/#resolvealias , this only matches the imports from module name